x-pack/plugin/esql/qa/testFixtures/src/main/resources/mapping-semantic_text.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I wonder if we can add a few more fields into the mapping, like `KEYWORD`, `TEXT`, some numeric types etc? The mapping looks a bit simple to me. If `SEMANTIC_TEXT` will be in the same category as `KEYWORD` and `TEXT`, we might want to support some functions that apply to them as well(perhaps in future PRs). It will be handy to have some extra fields in the same mapping to validate how  processing commands and functions work with the new data type, like eval/where/keep/drop/sort/stats/rename/mv_expand etc.

Reply:
done [e617196](https://github.com/elastic/elasticsearch/pull/113920/commits/e617196bfc36c52a17b3eb8bbc28c193924a4975).  More field types and functions can come later.